apps/web/src/app/globals.css

```
@@ -10,40 +10,54 @@
-  --sidebar-accent: oklch(0.9251 0.0071 88.645);
+  --sidebar-primary: oklch(0.6333 0.0309 154.9039);
+  --sidebar-primary-foreground: oklch(1.0000 0 0);
+  --sidebar-accent: oklch(0.9251 0.0071 88.6450);
   --sidebar-accent-foreground: oklch(0.2417 0.0298 269.8827);
-  --sidebar-border: oklch(0.9251 0.0071 88.645);
-  --sidebar-ring: oklch(0.6511 0.0826 258.0981);
-  --font-sans: var(--font-sans), ui-sans-serif, sans-serif, system-ui;
-  --font-serif: var(--font-serif), Georgia, serif;
+  --sidebar-border: oklch(0.9251 0.0071 88.6450);
+  --sidebar-ring: oklch(0.6333 0.0309 154.9039);
+  --font-sans: Antic, ui-sans-serif, sans-serif, system-ui;
```

Stylelint (17.4.0)

[error] Expected "Antic" to be "antic" (value-keyword-case)

(value-keyword-case)